cypress/support/step_definitions/index.ts

```diff
@@ -193,6 +193,10 @@ Then(/^the "(.*)" element contains "(.*)"$/, (selector: string, value: string) =
   $(selector).contains(value)
 })
 
+Then(/^the "(.*)" element has value "(.*)"$/, (selector: string, value: string) => {
+  $(selector).should('have.value', value)
+})
+
```

Comment:
Hm, technically that’s certainly possible I’d guess. In the `.then` call back we could check for the selected element’s tag name. On the other hand, I also think it’s nice to have it slightly differently phrased.

Reply:
It would look something like this:

```ts
Then(/^the "(.*)" element contains "(.*)"$/, (selector: string, value: string) => {
  $(selector).then(($el) => {
    const chainer = $el[0].tagName === 'INPUT' ? 'have.value' : 'contain'
    cy.wrap($el).should(chainer, value)
  })
})
```

Reply:
Changed this now.